repobee_sanitizer/_syntaxhelpers.py

```diff
@@ -0,0 +1,112 @@
+"""Functions and constants relating to the syntax of repobee-sanitizer.
+
+.. module:: _syntaxhelpers
+    :synopsis: Functions and constants relating to the syntax of
+    repobee-sanitizer.
+"""
+import pathlib
+from typing import List
+
+import re
+import repobee_plug as plug
+
+from repobee_sanitizer import _fileutils
+
+START_BLOCK = "REPOBEE-SANITIZER-START"
+REPLACE_WITH = "REPOBEE-SANITIZER-REPLACE-WITH"
+END_BLOCK = "REPOBEE-SANITIZER-END"
+
+SANITIZER_MARKERS = (
+    START_BLOCK,
+    REPLACE_WITH,
+    END_BLOCK,
+)
```

Comment:
Let's put these into an enum.

```suggestion
import enum

class Marker(enum.Enum):
    """The markers that define sanitizer blocks."""
    START = "REPOBEE-SANITIZER-START"
    REPLACE = "REPOBEE-SANITIZER-REPLACE-WITH"
    END = "REPOBEE-SANITIZER-END"
```
To get the string from one of those, you do the following:

```python
print(Marker.START.value) # prints REPOBEE-SANITIZER-START
```
Note that you must access the string with `.value`. It's not enough to just do `Marker.START`.

Also, we can iterate over the markers like so:

```python
for marker in Marker:
    print(marker.value) # prints the marker strings
````